### PR TITLE
Fortune: Switching to text template

### PR DESCRIPTION
**Pull request by mintsoft**
###### Description of changes

Ref: https://github.com/duckduckgo/zeroclickinfo-goodies/issues/2767

###### People to notify (@mention interested parties)

---

Instant Answer Page: https://duck.co/ia/view/fortune

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @frncscgmz

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: lib/DDG/Goodie/Fortune.pm [hunk 1]
```diff
@@ -18,12 +18,14 @@ handle remainder => sub {
...
 
-    return $output,
-      structured_answer => {
-        input     => [],
-        operation => 'Random fortune',
-        result    => $output
-      };
+    return $output, structured_answer => {
+        data => {
+            title => $output,
```
Comment (GuiltyDolphin) [inline lib/DDG/Goodie/Fortune.pm]: Subtitle should probably be `'Random fortune'`.

File: t/Fortune.t [hunk 2]
```diff
@@ -3,26 +3,31 @@
...
-my @fortune = (
-    '-ANY-',
-    structured_answer => {
-        input     => [],
-        operation => 'Random fortune',
-        result    => '-ANY-'
-    });
+sub build_test
+{
+    return test_zci(qr/.+/, structured_answer => {
```
Comment (GuiltyDolphin) [inline t/Fortune.t]: Again, we should wrap this in `re`.

**Comment by GuiltyDolphin** [comment]
@mintsoft Thanks!

**Pull request merged**